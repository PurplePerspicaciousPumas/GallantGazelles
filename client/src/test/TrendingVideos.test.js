import React from 'react';
import TrendingVideos from '../components/TrendingVideos.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <TrendingVideos />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});