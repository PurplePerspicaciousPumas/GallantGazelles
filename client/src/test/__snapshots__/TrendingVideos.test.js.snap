// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <table>
    <tbody>
      <tr>
        <td>
          <section>
            <div>
              <iframe
                allowfullscreen={true}
                frameborder="0"
                height="360"
                mozallowfullscreen={true}
                src="https://player.vimeo.com/video/206456987"
                webkitallowfullscreen={true}
                width="640"
              />
            </div>
          </section>
          <section>
            <div>
              <p>
                Shift
              </p>
              <p>
                Switch between Gmail accounts like a boss
              </p>
            </div>
          </section>
        </td>
        <td>
          <section>
            <div>
              <iframe
                allowfullscreen={true}
                frameborder="0"
                height="360"
                mozallowfullscreen={true}
                src="https://player.vimeo.com/video/206456987"
                webkitallowfullscreen={true}
                width="640"
              />
            </div>
          </section>
          <section>
            <div>
              <p>
                Shift
              </p>
              <p>
                Switch between Gmail accounts like a boss
              </p>
            </div>
          </section>
        </td>
      </tr>
      <tr>
        <td>
          <section>
            <div>
              <iframe
                allowfullscreen={true}
                frameborder="0"
                height="360"
                mozallowfullscreen={true}
                src="https://player.vimeo.com/video/206456987"
                webkitallowfullscreen={true}
                width="640"
              />
            </div>
          </section>
          <section>
            <div>
              <p>
                Shift
              </p>
              <p>
                Switch between Gmail accounts like a boss
              </p>
            </div>
          </section>
        </td>
        <td>
          <section>
            <div>
              <iframe
                allowfullscreen={true}
                frameborder="0"
                height="360"
                mozallowfullscreen={true}
                src="https://player.vimeo.com/video/206456987"
                webkitallowfullscreen={true}
                width="640"
              />
            </div>
          </section>
          <section>
            <div>
              <p>
                Shift
              </p>
              <p>
                Switch between Gmail accounts like a boss
              </p>
            </div>
          </section>
        </td>
      </tr>
    </tbody>
  </table>
</section>
`;
